### ISSUE TITLE
Script to compress the volume of JSON on S3

### ISSUE THREAD

**Issue by vschernoff** [protofire/filecoin-CID-checker#176]
We have a module s3-sort-caching within the Filecoin CID Checker that uploads MarketDeals to the S3. It is a simple Node.JS script that we run as a cronjob => https://github.com/protofire/filecoin-CID-checker/blob/calibration/packages/s3-sort-caching/index.js

The task is fairly simple. We need to start publishing the MarketDeals file that is 9GB in size now in a compressed way. As agreed with Eva, we will use .zstd archives as they are used in the Filecoin ecosystem and have a great compression ratio. 
If we would use bash, the command for compression would be: zstd --long -17 -T0
After some investigation DevOps team learnt that there is a ZSTD module for JS that might be of a help: https://www.npmjs.com/package/simple-zstd. Now we need a help from Node.JS developer that could incorporate this module into the solution. The overall estimation from our end is around 16 hours.

Consideration: 
As this is currently used in production - we need to make this in several steps. First - we add publishing of the compressed file IN PARALLEL with uncompressed. Then we notify the community about the new file and tell everyone to switch there. Then we disable publishing of the decompressed file after a while.

**Comment by koteld** [comment]
Task completed and merged into main and hyperspace branches.
